src/layout/Demo1.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import DemoHeader from '../components/Demo1/DemoOneHeader'
+
+import { DemoOneBackground } from '../components/Demo1/DemoOneBackground'
+import DemoOneBox from '../components/Demo1/DemoOneBox'
+//import { Context } from '../utils/context'
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+`
+
+export default function DemoOne(props) {
```

Comment:
Again please rename component without prefix of `DemoOne`. It's in a folder `Demo1`.

Reply:
Wouldn't it be easier to change the name of the folder??

Reply:
No, leave folder as itt's and update your components, we will keep that folder and add more for Demo2, Demo3 and so on.